tests/diagnostics/single-shader-record.slang.expected

```diff
@@ -0,0 +1,6 @@
+result code = -1
+standard error = {
+(0): error 39017: Can have at most one 'shader record' attributed constant buffer found 2.
```

Comment:
Style issues: diagnostic messages should use lower case (no leading capitalization), and stuff in single quotes should reflect the actual syntax a user writes (so, e.g., `’[[vk::shader_record]]’` instead of `’shader record’`). Also probably want either a semicolon before `found 2` or parentheses around it.

Longer term we’ll need to think about how to phrase this if we also allow entry-point parameters to (implicitly) define the shader record, since the terminology here won’t apply.